tic_tac_toe/game.py

```diff
@@ -87,18 +124,49 @@ def move(game, player, position):
     checks before the actual movement is done.
     After registering the movement it must check if the game is over.
     """
-    pass
-
+    if not _position_is_valid(position): 
+        raise InvalidMovement('Position out of range.')
+    if game['winner'] or _board_is_full(game['board']): #game is over
+        raise InvalidMovement('Game is over.')    
+    if not _position_is_empty_in_board(position, game['board']):
+        raise InvalidMovement('Position already taken.')
+    if not player == game['next_turn']:
+        raise InvalidMovement('"%s" moves next'%(game['next_turn']))
+    
+    
+    i = position[0]
+    j = position[1]
+    game['board'][i][j] = player
```

Comment:
Again this could be simplified to `game['board'][position[0]][position[1]] = player`